### PR TITLE
Basic doc improvments by going throught most of the articles

### PR DESCRIPTION
**Pull request by J12934**

### CODE REVIEW

File: docs/getting-started/installation.md [hunk 1]
```diff
@@ -31,26 +31,26 @@ Your now ready to install your [first scan types and start your first scans](/do
 
 The secureCodeBox supports the 4 latest Kubernetes releases (`v1.22`, `v1.21`, `v1.20` & `v1.19`). Older version might also work but are not officially supported or tested.
 
-To install the secureCodeBox we recommend using Helm at version 3.
+## Accessing the included MinIO Instance
 
-## Accessing the included Minio Instance
+The default secureCodeBox Operator included a [MinIO](https://min.io/) instance, which acts as a local S3 filestorage API used by the secureCodeBox to store the results files of its scans. You can switch it out with a S3 compatible API provided by most cloud providers.
```
Comment (malexmave) [inline docs/getting-started/installation.md:36]:
```suggestion
The default secureCodeBox Operator includes a [MinIO](https://min.io/) instance, which acts as a local S3 filestorage API used by the secureCodeBox to store the results files of its scans. You can switch it out with a S3 compatible API provided by most cloud providers.
```

File: docs/getting-started/first-scans.md [hunk 2]
```diff
@@ -200,24 +200,25 @@ Events:                      <none>
 ```
 
 This gives us an overview of the results of the scan.
-To view the actual findings produced by the scan you can use the download link to download the findings as JSON from Minio/S3.
+To view the actual findings produced by the scan you can use the download link to download the findings as JSON from MinIO/S3.
+
+:::note
+When using the default MinIO installation, you'll need to port-forward the to be able to access the finding download link. See: [installation](./installation#accessing-the-included-minio-instance)
```
Comment (malexmave) [inline docs/getting-started/first-scans.md:206]: Consistent link style

```suggestion
When using the default MinIO installation, you'll need to port-forward the to be able to access the finding download link. See: [installation](/docs/getting-started/installation#accessing-the-included-minio-instance)
```

File: docs/how-tos/automatically-repeating-scans.md [hunk 3]
```diff
@@ -8,13 +8,12 @@ title: "Repeating Scans on a Schedule"
 
 ## Introduction
 
-In this step-by-step tutorial, we will go through all the required stages to set up a repeating scan with the secureCodeBox. A Repeating scan will run automatically each time, a time interval is passed. This time interval is set by the user.  
-In this example, we are going to run a repeating typo3scan scan on the Typo3 local vulnerable demo-target (old-typo3).   
-A repeating scan is useful, as it allows the developer to be aware of any new vulnerabilities that have been introduced in development.   
+In this step-by-step tutorial, we will go through all the required stages to set up a repeating scan with the secureCodeBox. A repeating scan will run automatically each time, a time interval is passed. This time interval is set by the user. In this example, we are going to run a repeating typo3scan scan on the Typo3 local vulnerable demo-target (old-typo3). A repeating scan is useful, as it allows the developer to be aware of any new vulnerabilities that have been introduced in development.
```
Comment (malexmave) [inline docs/how-tos/automatically-repeating-scans.md:11]:
```suggestion
In this step-by-step tutorial, we will go through all the required stages to set up a repeating scan with the secureCodeBox. A repeating scan will run automatically each time a time interval is passed. This time interval is set by the user. In this example, we are going to run a repeating typo3scan scan on the Typo3 local vulnerable demo-target (old-typo3). A repeating scan is useful, as it allows the developer to be aware of any new vulnerabilities that have been introduced in development.
```

File: docs/how-tos/scanning-networks.md [hunk 4]
```diff
@@ -98,7 +98,7 @@ kubectl delete scans --all
...
+Now we also want to start the **Ncrack scan** after Nmap has finished. We use [cascading rules](/docs/hooks/cascading-scans/) for that. Cascading rules allow us to automatically call any scanner that matches our given labels (see below) in order to start further security checks. In this case, we want to follow up on with an ncrack scan to test the credentials configured for the server. Fortunately, the secureCodeBox already comes with a [predefined ncrack cascading rule] for ssh scanning. We will take an in-depth look at the cascading rule to understand what's happening:
```
Comment (malexmave) [inline docs/how-tos/scanning-networks.md:101]: What is the "predefined ncrack cascading rule" supposed to link to? Docs or source code? It does not give a link target.

**Review by malexmave — COMMENTED** [review]
A few small things, then we can merge this and make a new PR for the remaining documentation changes.

File: docs/how-tos/scanning-networks.md (hunk 4 above)
Comment (J12934) [reply docs/how-tos/scanning-networks.md:101]: Acutal url is down at the bottom of the file: 

https://github.com/secureCodeBox/documentation/blob/5f5044ee3da37006decdf4df892fd8d9b426a9a4/docs/how-tos/scanning-networks.md?plain=1#L200

Comment (malexmave) [reply docs/how-tos/scanning-networks.md:101]: Oh, right, that syntax for links works too. Nevermind then.

**Review by malexmave — APPROVED** [review]

File: docs/how-tos/scanning-networks.md (hunk 4 above)
Comment (J12934) [reply docs/how-tos/scanning-networks.md:101]: yeah...
I'm also not a big fan of that syntax :)

**Pull request merged**